### PR TITLE
MLR-496: Remove vestiges of multiple district code alerting in services and UI

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: docker/secrets.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-oauthClientSecret=mlr-secret
+oauthClientSecret=changeMe
```
Comment (zmoore-usgs) [inline docker/secrets.env:1]: This new value is what the KeyCloak version of local dev expects

Comment (kmschoep-usgs) [reply docker/secrets.env]: Ah, yep.  It was a bit of a merge 💩 show over here!

Comment (zmoore-usgs) [reply docker/secrets.env]: Yeah sorry about that.. 😅

**Comment by dpattermann-usgs** [comment]
Wait, are we getting rid of the parseWorkflow?

**Comment by kmschoep-usgs** [comment]
Yes, we only needed it for the popups, and we're getting rid of the popups and replacing them with a static message in the UI.

**Pull request merged**